src/routes/api/instances/labels/predictions/+server.ts

```diff
@@ -0,0 +1,15 @@
+import type { Tables } from '$lib/supabase.js';
+import { error } from '@sveltejs/kit';
+
+export async function POST({ locals: { supabase }, request }) {
```

Comment:
It's early but I'm confused - this is just updating instances, not predictions? why is it in predictions path? it updates labels right?